### PR TITLE
Porting from previous PR.

### PR DESCRIPTION
**Pull request by lleonard-msft**
This PR replaces PR #2667.  

(The Build TOC changes led to a merge conflict that would've taken longer to resolve than it took to create a new PR.)

### CODE REVIEW

File: api-reference/beta/resources/azure_ad_auditlog_overview.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Azure AD audit log API overview
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Microsoft Azure Active Directory (Azure AD) tracks user activity and sign-on metrics and creates audit log reports that help you understand how your users access and leverage Azure AD services. Use the Microsoft Graph API for Azure AD to analyze the data underlying these reports and to create custom solutions tailored to your organization's specific needs.
```
Comment (dkershaw10) [inline api-reference/beta/resources/azure_ad_auditlog_overview.md]: Not sure this should say "Microsoft"

Comment (dkershaw10) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Also inconsistent terminology.  "sign-on" here and "sign-in" below.  I think we use the latter.

File: api-reference/beta/resources/azure_ad_auditlog_overview.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Audit logs
+
+The audit logs activity report provides you with access to the history of every task performed in your tenant. The audit logs report provides you with records of system activities for compliance. Amongst others, the provided data enables you to address common scenarios such as:
+
+- Someone in my tenant got access to an admin group. Who gave them access? 
+
+- I want to know which users are signing into a recently on-boarded app and I want to know if it's doing well.
+
+- How passwords resets have there been within the tenant?
```
Comment (dkershaw10) [inline api-reference/beta/resources/azure_ad_auditlog_overview.md]: How many

File: api-reference/beta/resources/azure_ad_auditlog_overview.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- audit logs 
+- sign-in logs
+
+### Audit logs
+
+The audit logs activity report provides you with access to the history of every task performed in your tenant. The audit logs report provides you with records of system activities for compliance. Amongst others, the provided data enables you to address common scenarios such as:
+
+- Someone in my tenant got access to an admin group. Who gave them access? 
+
+- I want to know which users are signing into a recently on-boarded app and I want to know if it's doing well.
```
Comment (dkershaw10) [inline api-reference/beta/resources/azure_ad_auditlog_overview.md]: signing in to

Comment (dkershaw10) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: ...recently acquired app...

File: api-reference/beta/resources/azure_ad_auditlog_overview.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Azure AD provides two types of activity logs:
+
+- audit logs 
+- sign-in logs
+
+### Audit logs
+
+The audit logs activity report provides you with access to the history of every task performed in your tenant. The audit logs report provides you with records of system activities for compliance. Amongst others, the provided data enables you to address common scenarios such as:
+
+- Someone in my tenant got access to an admin group. Who gave them access?
```
Comment (dkershaw10) [inline api-reference/beta/resources/azure_ad_auditlog_overview.md]: Do we use tenant or directory?

File: api-reference/beta/resources/azure_ad_auditlog_overview.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Here are popular requests for working with audit log data:
+
+Operation | URL
+:----------|:----
+GET tenant user activities | [https://graph.microsoft.com/beta/auditLogs/directoryAudits](https://developer.microsoft.com/graph/graph-explorer?request=auditLogs/directoryAudits&version=beta)
+GET tenant user sign-ins | [https://graph.microsoft.com/beta/auditLogs/signIns](https://developer.microsoft.com/graph/graph-explorer?request=auditLogs/signIns&version=beta)
+
+## What licenses do I need?
+
+Audit log reports are available for features that you've licensed.  If you have a license for a specific feature, you also have access to its audit logs.
```
Comment (dkershaw10) [inline api-reference/beta/resources/azure_ad_auditlog_overview.md]: Be good to a) have an example like - you need Azure AD Premium P1 to access *self-service password reset* audit reports - or some such, and b) link to the Azure AD licensing page - https://azure.microsoft.com/en-us/pricing/details/active-directory/

Comment (lleonard-msft) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Updated.

File: api-reference/beta/resources/azure_ad_auditlog_overview.md (hunk 4 above)
Comment (lleonard-msft) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Unsure; will changes and coordinate offline.  Changed for now.

File: api-reference/beta/resources/azure_ad_auditlog_overview.md (hunk 3 above)
Comment (lleonard-msft) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Both fixed.

File: api-reference/beta/resources/azure_ad_auditlog_overview.md (hunk 2 above)
Comment (lleonard-msft) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Fixed.

File: api-reference/beta/resources/azure_ad_auditlog_overview.md (hunk 1 above)
Comment (lleonard-msft) [reply api-reference/beta/resources/azure_ad_auditlog_overview.md]: Both fixed.

**Comment by dkershaw10** [comment]
@lleonard-msft LGTM. Merge!

**Pull request merged**